socialDistribution/app/static/app/js/profile.js

```diff
@@ -183,7 +183,7 @@ async function getSingleAuthorInfo(url, token){
 
     if (token){
         headers = new Headers({
-                'Authorization': 'Basic '+token, 
+                'Authorization': 'Basic '+token,
```

Comment:
Are these meant to be part of these change?

Reply:
Oh scratch this, looks to be just a space change

Reply:
my vs code likes to remove spaces lol